### PR TITLE
Content Update

### PR DESCRIPTION
**Pull request by kelvinkipruto**
Updated content as per the new design

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { CustomRouteReuseStrategy } from './app-routing.module';
 import { OauthApiService } from './oauth_api.service';
 import { focusElement } from './common/a11y_utils';
 
-const APP_TITLE = 'Harassment Manager';
+const APP_TITLE = 'SafetyNet';
```
Comment (kilemensi) [inline src/app/app.component.ts:25]:
```suggestion
const APP_TITLE = 'SafeNet';
```

Lets use SafeNet everywhere.

File: src/app/privacy-page/privacy-page.component.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<div class="info-section ">
```
Comment (kilemensi) [inline src/app/privacy-page/privacy-page.component.html:1]: Isn't this a full standalone page? I expected us to either use find-support or help-center as templates.

**Review by kilemensi — COMMENTED** [review]
👍🏽 @kelvinkipruto 

----

1. Is it possible to just edit content without formatting? I see a lot of changes that are just formatting.
2. I see package-lock.json is being updated. Is there any reason? If not, use something like volta to install the exact node version needed that won't update this file.

**Comment by kelvinkipruto** [comment]
@kilemensi Content Updated. We can now try to update the ui.

**Comment by kilemensi** [comment]
> @kilemensi Content Updated. We can now try to update the ui.

Not sure I follow @kelvinkipruto. Have you already addressed my feedback: https://github.com/CodeForAfrica/harassment-manager/pull/2#pullrequestreview-1103912732?

**Comment by kelvinkipruto** [comment]
@kilemensi Sorry for the confusion. I made that comment before reading your feedback. Let me address them.

File: src/app/privacy-page/privacy-page.component.html (hunk 2 above)
Comment (kelvinkipruto) [reply src/app/privacy-page/privacy-page.component.html:1]: The doc contained content for the privacy page, but that page did not exist, and was marked as [todo](https://github.dev/CodeForAfrica/harassment-manager/blob/main/src/app/footer/footer.component.html#L53-L54) so we have to create it.

Comment (kilemensi) [reply src/app/privacy-page/privacy-page.component.html:1]: Yes @kelvinkipruto. I meant I expected us to follow either [`help-center`](https://github.com/CodeForAfrica/harassment-manager/tree/main/src/app/help-center) or [`find-support`](https://github.com/CodeForAfrica/harassment-manager/tree/main/src/app/find-support) structure when creating the new page. Looking at the code on this PR, are we following any of the existing page structure?

File: src/app/app.component.ts (hunk 1 above)
Comment (kelvinkipruto) [reply src/app/app.component.ts:25]: Let me update it

**Comment by kelvinkipruto** [comment]
Regarding the package-lock.json, let me try using volta.

File: src/app/privacy-page/privacy-page.component.html (hunk 2 above)
Comment (kelvinkipruto) [reply src/app/privacy-page/privacy-page.component.html:1]: Yes, now I get you. Let me update it.

File: src/app/privacy-page/privacy-page.component.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
+<div class="header">
+    <h1>Privacy Policy</h1>
+    <div class="subtitle">
+        Code for Africa is committed to protecting your personal data. This Privacy Policy will inform you as to how
+        we look after your personal data when you visit <a href="/">SafetyNet</a> or use the SafetyNet tool and tell you
```
Comment (kilemensi) [inline src/app/privacy-page/privacy-page.component.html:5]:
```suggestion
        we look after your personal data when you visit <a href="/">SafeNet</a> or use the SafeNet tool and tell you
```

File: src/app/footer/footer.component.html [hunk 4]
```diff
@@ -51,7 +51,7 @@
 
   <div class="bottom-row">
     <!-- TODO: Provide links. -->
-    <a href="" target="_blank">Privacy</a>
+    <a href="/privacy-policy">Privacy</a>
     <a href="" target="_blank">Terms of Service</a>
```
Comment (kilemensi) [inline src/app/footer/footer.component.html:55]: Make terms also open the same `/privacy-policy` page for now.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

The description is supposed to be limited in width similar to other pages
Current
Screenshot 2022-09-14 at 13-25-09 Privacy Policy - SafeNet
Should be like
Screenshot 2022-09-14 at 13-25-21 Find Support - SafeNet

----

We should have this deployed in the next hour.

**Review by kilemensi — APPROVED** [review]
🚀 

---

Lets deploy this.

**Pull request merged**